src/components/tabbed-list/tabbed-list-item.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import format from "date-fns/format";
+import "./tabbed-list.scss";
+import CaretSvg from "../../icons/caret.svg";
+
+export const TabbedListItem = ({
+  color,
+  date,
+  resultText,
+  symptomText,
+  onClick
+}) => {
+  return (
+    <div className={"tli-container"}>
+      <div className={"tli-date"}>{format(date, "dd.MM.yyyy")}</div>
+      <div className={"tli-text-wrapper"} onClick={onClick}>
+        <div className={"tli-text"} onClick={onClick}>
```

Comment:
can we trigger the onClick only when clicking on the blue/red button? I think it's unexpected that when I select text the alert pops up (also for some reason it pops up twice when i click on a list item)